src/Http/Routing/ref/Microsoft.AspNetCore.Routing.netcoreapp3.0.cs

```diff
@@ -51,6 +44,10 @@ public static partial class RoutingBuilderExtensions
     public static partial class RoutingEndpointConventionBuilderExtensions
     {
         public static Microsoft.AspNetCore.Builder.IEndpointConventionBuilder RequireHost(this Microsoft.AspNetCore.Builder.IEndpointConventionBuilder builder, params string[] hosts) { throw null; }
+        public static Microsoft.AspNetCore.Builder.IEndpointConventionBuilder WithDisplayName(this Microsoft.AspNetCore.Builder.IEndpointConventionBuilder builder, System.Func<Microsoft.AspNetCore.Builder.EndpointBuilder, string> func) { throw null; }
+        public static Microsoft.AspNetCore.Builder.IEndpointConventionBuilder WithDisplayName(this Microsoft.AspNetCore.Builder.IEndpointConventionBuilder builder, string displayName) { throw null; }
+        public static Microsoft.AspNetCore.Builder.IEndpointConventionBuilder WithMetadata(this Microsoft.AspNetCore.Builder.IEndpointConventionBuilder builder, System.Collections.Generic.IEnumerable<object> items) { throw null; }
+        public static Microsoft.AspNetCore.Builder.IEndpointConventionBuilder WithMetadata(this Microsoft.AspNetCore.Builder.IEndpointConventionBuilder builder, params object[] items) { throw null; }
```

Comment:
Are we worried about the params object array overload conflicting withe the object IEnumerable overload?

https://stackoverflow.com/questions/15933632/passing-array-to-function-that-takes-either-params-object-or-ienumerablet

Reply:
yeah that's a good point. I think having the params version is more valuable